docs/source/models/cause.rst

```diff
@@ -558,6 +558,113 @@ below.
 Remission Rates
 ^^^^^^^^^^^^^^^
 
+**Definition**
+
+Remission is a measure of cases that recover from a with-condition state, given 
+a specified population and time period. Just as with incidence, the numerator is 
+given by the count of recovered (or *remitted*) cases, and the denominator is 
+the cumulative person-time [link to person-time definition in incidence 
+section] during which cases are able to go into remission.
+
+  For example, consider diarrhea cases in the Philippines in 2017. Say there
+  were 53,362,490 prevalent cases of diarrhea over the course of 2017, and each
+  case remitted after an average of 5 days. Note that here we are using *period
+  prevalence*; however, GBD (and thus we) typically mean *point prevalence* when 
+  we refer to prevalence. Then, the way we then define this remission rate 
+  depends on the denominator we choose. Some examples include:
+
+  .. math::
+
+    \frac{\text{1/5 case}}{\text{1 person-days}}=\frac{365/5\text{ cases}}{\text{1 person-year}}=73\text{ cases/person-year}
```

Comment:
2 comments:

1. It might be clearer to say (1 case) / (5 person-days) rather than putting the fraction 1/5 in the numerator.
2. Since the paragraph starts with the period prevalence of diarrheal diseases, I was expecting that number to show up in the calculation, but this calculation is independent of that number - it just uses the average duration. It would be less confusing if you didn't mention the period prevalence at all (or introduce it later if you need it for something else).

Reply:
1. Sounds good

2. Yes! I think I backed myself into a corner with point/period prevalence, when it isn't entirely relevant here. I've simplified the whole section down.

Reply:
I like the simplified version much better!